src/RazorSdk/Targets/Microsoft.NET.Sdk.Razor.SourceGenerators.targets

```diff
@@ -16,13 +16,14 @@ Copyright (c) .NET Foundation. All rights reserved.
 
     <PropertyGroup>
       <_RazorSdkSourceGeneratorDirectoryRoot>$(RazorSdkDirectoryRoot)\source-generators\</_RazorSdkSourceGeneratorDirectoryRoot>
+      <_RazorSdkTasksDirectoryRoot>$(RazorSdkDirectoryRoot)\tasks\net6.0\</_RazorSdkTasksDirectoryRoot>
     </PropertyGroup>
 
     <!-- Configure analyzers -->
     <ItemGroup>
-      <_RazorAnalyzer Include="$(_RazorSdkSourceGeneratorDirectoryRoot)Microsoft.AspNetCore.Mvc.Razor.Extensions.dll" />
-      <_RazorAnalyzer Include="$(_RazorSdkSourceGeneratorDirectoryRoot)Microsoft.AspNetCore.Razor.Language.dll" />
-      <_RazorAnalyzer Include="$(_RazorSdkSourceGeneratorDirectoryRoot)Microsoft.CodeAnalysis.Razor.dll" />
+      <_RazorAnalyzer Include="$(_RazorSdkTasksDirectoryRoot)Microsoft.AspNetCore.Mvc.Razor.Extensions.dll" />
```

Comment:
I think there's a bunch of extra files in the tasks directory compared to the 5.0 SDK. For e.g, 

```
ls 'C:\Program Files\dotnet\sdk\5.0.103\Sdks\Microsoft.NET.Sdk.Razor\tasks\net5.0'


    Directory: C:\Program Files\dotnet\sdk\5.0.103\Sdks\Microsoft.NET.Sdk.Razor\tasks\net5.0


Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
-a----         1/23/2021  12:33 PM          76680 Microsoft.NET.Sdk.Razor.Tasks.dll
```

vs 
```
 ls D:\temp\dotnet6\sdk\6.0.100-preview.3.21124.4\Sdks\Microsoft.NET.Sdk.Razor\tasks\net6.0\


    Directory: D:\temp\dotnet6\sdk\6.0.100-preview.3.21124.4\Sdks\Microsoft.NET.Sdk.Razor\tasks\net6.0

Mode                LastWriteTime         Length Name
----                -------------         ------ ----
d-----         2/24/2021 10:32 AM                cs
d-----         2/24/2021 10:32 AM                de
d-----         2/24/2021 10:32 AM                es
d-----         2/24/2021 10:32 AM                fr
d-----         2/24/2021 10:32 AM                it
d-----         2/24/2021 10:32 AM                ja
d-----         2/24/2021 10:32 AM                ko
d-----         2/24/2021 10:32 AM                pl
d-----         2/24/2021 10:32 AM                pt-BR
d-----         2/24/2021 10:32 AM                ref
d-----         2/24/2021 10:32 AM                ru
d-----         2/24/2021 10:32 AM                tr
d-----         2/24/2021 10:32 AM                zh-Hans
d-----         2/24/2021 10:32 AM                zh-Hant
-a----         2/24/2021  1:15 PM         156672 Microsoft.AspNetCore.Mvc.Razor.Extensions.dll
-a----         2/24/2021  1:15 PM        2572288 Microsoft.AspNetCore.Razor.Language.dll
-a----         2/24/2021  1:15 PM       15308800 Microsoft.CodeAnalysis.CSharp.dll
-a----         2/24/2021  1:15 PM        6086656 Microsoft.CodeAnalysis.dll
-a----         2/24/2021  1:15 PM         174592 Microsoft.CodeAnalysis.Razor.dll
-a----         2/24/2021  1:15 PM         167424 Microsoft.NET.Sdk.Razor.Tasks.dll
```

I would have expected we would to pick up Razor.Language and co from the `tools` directory.